ReactAndroid/src/main/java/com/facebook/react/devsupport/JSDebuggerWebSocketClient.java

```diff
@@ -218,14 +215,12 @@ public void onMessage(ResponseBody response) throws IOException {
       } else {
         abort("Parsing response message from websocket failed", e);
       }
-    } finally {
-      response.close();
     }
   }
 
   @Override
-  public void onFailure(IOException e, Response response) {
-    abort("Websocket exception", e);
+  public void onFailure(WebSocket webSocket, Throwable t, Response response) {
+      t.printStackTrace();
```

Comment:
This appears to be a change in behavior in the failure case. Is that intentional?